### ISSUE TITLE
Update permission storage

### ISSUE THREAD

**Issue by AlmightyYakob** [multinet-app/multinet-server#456]
Creating a new issue, since this wont' be addressed in #455.

Currently, we store permissions by storing the `sub` value of users. However, for use by the client, the entire user object is necessary. The idea of this is to store the entire user object in the permissions of a workspace, so it's both available to the client, and doesn't take extra time to transform.

However, I'm now realizing that there are some issues in this approach. For one, there'd now be a potential for data mismatch, between the users collection, and the user object stored in a workspace's permissions. In this case we may want to either keep the response transformation, or create an endpoint where the client can use the `sub` values to retrieve the entire user.

I'm not sure what the right answer here is, just thinking about the possibilities. If we decide not to change the permission storage, this issue should be closed.

**Comment by waxlamp** [comment]
For now, I want to avoid denormalizing in this way.

If the issue is performance, I believe the proper solution for this is instead to look at creating Foxx microservices to send back composed, structured data as needed by the API. Once the ORM layer lands, we can take stock of how it affects performance, and then we can begin thinking about how to add methods to the ORM classes that more closely match the API needs, and provide corresponding Foxx microservices that produce the data in the shape needed, with a boost in performance.

This will take some discussion, thinking, and writing. Let's leave this issue open for now, until we reach that stage.
